include/SDL3/SDL_gamepad.h

```diff
@@ -174,7 +174,7 @@ typedef struct SDL_GamepadBinding
  *
  * \param mappingString the mapping string
  * \returns 1 if a new mapping is added, 0 if an existing mapping is updated,
- *          -1 on error; call SDL_GetError() for more information.
+ *          negative error code; call SDL_GetError() for more information.
```

Comment:
` or a negative error code on failure; call SDL_GetError() for more information.`